client-lib/src/main/java/com/signnow/library/dto/Document.java

```diff
@@ -38,6 +50,75 @@ public SigningLinkRequest(String documentId) {
         }
     }
 
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    public static class Thumbnail {
+        public String small;
+        public String medium;
+        public String large;
+    }
+
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    public static class Signature {
+        public String id;
+        @JsonProperty("user_id")
+        public String userId;
+        public String email;
+        @JsonProperty("page_number")
+        public String pageNumber;
+        public String width;
+        public String height;
+        public String x;
+        public String y;
+        public String created;
+        public String data;
+    }
+
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    public static class DocumentField {
+        public String id;
+        public FieldType type;
+        @JsonProperty("role_id")
+        public String roleId;
+        public String role;
+        public String originator;
+        public String fulfiller;
+        @JsonProperty("field_request_id")
+        public String fieldRequestId;
+        @JsonProperty("element_id")
+        public String elementId;
+        @JsonProperty("field_request_canceled")
+        public String fieldRequestCanceled;
+        @JsonProperty("template_field_id")
+        public String fieldTemplateId;
+        @JsonProperty("field_id")
+        public String fieldId;
+        @JsonIgnore
+        public Map<String, String> attributes = new HashMap<>();
```

Comment:
Extend such classes form GenericId, it already contains this routines

Reply:
done